Mithril update 2025-01-08 (#521)
IntersectMBO/cardano-updates: blog/2025-01-08-mithril.md

---
title: Mithril Team Update
slug: 2025-01-08-mithril
authors: jpraynaud
tags: [mithril]
hide_table_of_contents: false
---

## High level overview

> The Mithril team released the new distribution [`2450.0`](https://github.com/input-output-hk/mithril/releases/tag/2450.0). This distribution offers stable support for Cardano node v.10.1.3, and it includes bug fixes and performance improvements.
> 
> The team continued implementing the incremental certification of the Cardano database: they kept working on creating and synchronizing the artifacts, they implemented the artifact routes and they started working on the digests route for the incremental Cardano database in the aggregator REST API.
> 
> Finally, the team completed the experimental cache for certificate verification in the WASM client, they implemented enhancements to the CI, and they fixed a bug with identifier collisions in the incremental Cardano database artifacts.

Reviewer: ```suggestion
The Mithril team released the new distribution [`2450.0`](https://github.com/input-output-hk/mithril/releases/tag/2450.0), which offers stable support for Cardano node v.10.1.3 and includes bug fixes and performance improvements.

The team also continued implementing the incremental certification of the Cardano database: they kept working on creating and synchronizing the artifacts, implemented the artifact routes, and started working on the digest route for the incremental Cardano database in the aggregator REST API.

Finally, they completed the experimental cache for certificate verification in the WASM client, implemented enhancements to the CI, and fixed a bug with identifier collisions in the incremental Cardano database artifacts.
```
